pkg/arvo/sys/vane/ames.hoon

```diff
@@ -1909,6 +1909,9 @@
       =/  =bone  bone.shut-packet
       ::
       ?:  ?=(%& -.meat.shut-packet)
+        =+  ?~  dud  ~
+            %.  ~
+            (slog leaf+"ames: crashed on fragment" >mote.u.dud< tang.u.dud)
```

Comment:
I though we were going to print one line by default, and put the whole trace behind I flag -- do you think this happens rarely enough to print the full trace unconditionally?